awx/main/utils/__init__.py

```diff
@@ -2,5 +2,8 @@
 # All Rights Reserved.
 
 # AWX
-from awx.main.utils.common import * # noqa
-from awx.main.utils.encryption import * # noqa
+from awx.main.utils.common import *  # noqa
```

Comment:
yeah... I expected this would have to get left.

I remember when we moved from the `utils.py` file (IIRC) to a folder. We could still stand to break those up into more files. Like, there's a lot related to activity stream and serialization. And there's a bagillion in-line imports because there are so many things doing so many different conflicting things.

Reply:
Yea, it'd be nice to split those up some more at some point.  That and `views.py`, which is a monster.